sdk/communication/azure-communication-chat/README.md

```diff
@@ -15,7 +15,9 @@ This package contains the Chat client library for Azure Communication Services.
 * The library is written in Java 8. Your application must be built with Android Gradle plugin 3.0.0 or later, and must be configured to [enable Java 8 language desugaring](https://developer.android.com/studio/write/java8-support.html#supported_features) to use this library. Java 8 language features that require a target API level > 21 are not used, nor are any Java 8+ APIs that would require the Java 8+ API desugaring provided by Android Gradle plugin 4.0.0.
 
 ### Versions available
-The current version of this library is **1.0.0-beta.2**.
+The current Server - Chat Api Version is **2020-11-01-preview3**.
+
+The current Client - Chat SDK Version is **1.0.0-beta.3**.
```

Comment:
Maybe we can rephrase this to:
```suggestion
The current Azure Communication Chat Service Version is **2020-11-01-preview3**.

The current Azure Communication Chat SDK Version is **1.0.0-beta.3**.
```
Or some other way that makes it easier to read.